views/stats.ejs

```diff
@@ -1140,11 +1150,15 @@ const metaDescription = getMetaDescription()
           <% } %>
           <% if(calculated.profile.game_mode == 'ironman'){ %>
             <% if(notAvailable.length > 0){ %><br><br><% } %>
-            This is an <strong>Ironman</strong> profile. The player cannot use the auction house, bazaar, trade or pick up drops from other players.
+            This is an <strong>Ironman</strong> profile. The player cannot use the auction house, bazaar, trade, or pick up drops from other players.
           <% } %>
           <% if(calculated.profile.game_mode == 'bingo'){ %>
             <% if(notAvailable.length > 0){ %><br><br><% } %>
-            This is a <strong>Bingo</strong> profile. The player cannot spend gems, use the auction house, bazaar, trade or pick up drops from other players.
+            This is a <strong>Bingo</strong> profile. The player cannot spend gems, use the auction house, bazaar, trade, or pick up drops from other players.
+          <% } %>
+          <% if(calculated.profile.game_mode == 'island'){ %>
+            <% if(notAvailable.length > 0){ %><br><br><% } %>
+            This is a <strong>Stranded</strong> profile. The player cannot leave their skyblock island or trade with other players.
```

Comment:
is trading with other players really needed

Reply:
I had the same though but then realised people can still /visit stranded gamemode players so... I guess it's good to specify that

Reply:
I think so because other people can still visit their island